applications/ray/main.tf

```diff
@@ -157,3 +154,17 @@ module "kuberay-cluster" {
   depends_on             = [module.gcs, module.kuberay-operator]
 }
 
+
+# Assign resource quotas to Ray namespace to ensure that they don't overutilize resources
```

Comment:
Thinking of taking resource_quota out of modules from rag/ray/jupyter and define a new module.. that will be invoked during namespace creation module or after that. 

these quotas are assigned early and are not configured at app module level.. (also since its namespace level config, it make more sense outside of app modules)

WDYT?

Reply:
I don't see what benefit this gives us. Ultimately, we're still implementing a ResourceQuota on a namespace in a particular application. By wrapping this resource in a module, we're simply adding an additional layer of indirection that requires more work to plumb through options. It made sense on the Ray module since my goal is to harden the Ray cluster but I was fine pulling it up to the Ray application to try to make progress.